pinot-controller/src/test/java/com/linkedin/pinot/controller/helix/core/realtime/SegmentCompletionTest.java

```diff
@@ -320,6 +320,28 @@ public void testHappyPath(long startTime) throws  Exception {
   }
 
   @Test
+  public void testWinnerOnTimeLimit() throws Exception {
+    SegmentCompletionProtocol.Response response;
+    Request.Params params;
+    segmentCompletionMgr._secconds = 10L;
```

Comment:
Yes, the unit test sets them in seconds. This is mock time, btw. You can read the other tests to get an idea.